<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculatrice Matricielle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="SaisieMatrice">
        <property name="text">
         <string>Créer une nouvelle matrice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gerermatrice">
        <property name="text">
         <string>Gérer les matrices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="enregisterMatriceFichier">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>Enregistrer une matrice dans un fichier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ouvrireMatriceFichier">
        <property name="text">
         <string>Ouvrir une matrice depuis un fichier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="faireOperation">
        <property name="text">
         <string>Faire une opération entre deux matrices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="multMatriceScal">
        <property name="text">
         <string>Multiplier une matrice par un scalaire</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calculerDeterminant">
        <property name="text">
         <string>Calculer un déterminant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="determinerMatriceInverse">
        <property name="text">
         <string>Déterminer une matrice inverse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calculerTranspose">
        <property name="text">
         <string>Calculer une transposée</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calculerPuissance">
        <property name="text">
         <string>Calculer une puissance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calculerNorme">
        <property name="text">
         <string>Calculer une norme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calculerTrace">
        <property name="text">
         <string>Calculer une trace</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
